graph.py

```diff
@@ -0,0 +1,81 @@
+from linkedlist import LinkedList
+
+
+class Graph:
+    def __init__(self):
+        self._storage = {}
+
+    def nodes(self):
+        """Returns a list of all the nodes in the graph/"""
```

Comment:
It's interesting that you have chosen to start writing doc strings...

If you are gong to write them please read this https://peps.python.org/pep-0257/

All of these are written as descriptions but should be commands. Returns --> Return

Reply:
I didn't mean to leave that. thank you!

Reply:
Bump. Fix these Doc strings or remove them.  Fixing them is preferred so you learn what a doc string should look like, whether you choose to use them or not.